.readthedocs.yml

```diff
@@ -3,10 +3,10 @@
 version: 2
 
 build:
-  os: ubuntu-22.04
+  os: ubuntu-24.04
   tools:
     python: >-
-      3.9
+      3.12
```

Comment:
N.B. In the past, I'd keep this at the lowest possible value to accommodate docs contributors running this on various versions. However, nowadays, I sometimes make it have the version that matches GHA.

Reply:
@bdraco this is something I'd like to have in the future: https://github.com/ansible/awx-plugins/blob/8ba56cc/.readthedocs.yaml — the same entry-points/automations being called @ RTD and GHA. That project doesn't use make, but I wanted to show the general idea.

Reply:
That does looks like a nicer / more maintainable design.  I'm still wrapping my head around all the GHAs between all the projects.  Its a lot to take in, and I'm playing catch up.